src/export/StructureDefinitionExporter.ts

```diff
@@ -162,7 +162,15 @@ export class StructureDefinitionExporter implements Fishable {
               const instanceExporter = new InstanceExporter(this.tank, this.pkg, this.fisher);
               const instance = instanceExporter.fishForFHIR(rule.value as string);
               if (instance == null) {
-                logger.error(`Cannot find definition for Instance: ${rule.value}. Skipping rule.`);
+                if (element.type?.length === 1) {
+                  logger.error(
+                    `Cannot assign Instance at ${rule.value} to element of type ${element.type[0].code}. Instance definition not found.`
```

Comment:
I feel like I find the phrase "Instance at ${rule.value}" a little confusing. I think "at" made me thing it would be referring to a path. Maybe we could say 

```
`Cannot assign Instance at path ${rule.path} to element of type ${element.type[0].code}. Definition not found for Instance: ${rule.value}.`
```

Although, now I can't tell if that's too wordy or if it even is clearer.

Reply:
I tried to clarify the error message in https://github.com/FHIR/sushi/pull/714/commits/61139c3c727b7b1bf5905c511a6d7cdb0ec64aa7. The idea of this message is just that having the type of the element is helpful for people who may be trying to assign a Codeable, but are accidentally using the syntax to assign an instance.